backend/src/test/java/gov/cdc/usds/simplereport/db/model/FeatureFlagTest.java

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FeatureFlagTest {
+  FeatureFlag featureFlag = new FeatureFlag();
+
+  @Test
+  void equals_sameObject_returnsTrue() {
+    assertEquals(featureFlag, featureFlag);
+  }
```

Comment:
Is this just testing the lombok annotations? If yes, is this done for sonarcloud?

Reply:
I just added that one because I saw it in other test classes for other Entity/Model Objects 🤷‍♀️ . I don't have a good answer... I was just trying to follow the pattern.

Reply:
I think as long as it doesn't impact sonarcloud (not sure how to check actually) it should be ok to remove I think. I don't see a ton of value in testing Lombok annotations.  

If this is a pattern the team does follow regularly, then I'm ok with this.

Reply:
@johanna-skylight just for your SA, I think this is only done in other classes to test that overriding the `.equals` of a given object is working as intended. So we don't need it here because you're not changing the default equality.